src/btrfs/fs.rs

```diff
@@ -86,7 +86,7 @@ pub struct BtrfsIoctlSearchHeader {
     pub len: u32,
 }
 
-enum FsInner {
+pub enum FsInner {
```

Comment:
See above comment -- should be private imo

Reply:
Thanks, fixed in d6685d0.